src/config.ts

```diff
@@ -10,3 +11,5 @@ export const SUPPORTED_METHODS = {
 };
 
 export const WS_PORT = 8000;
+
+export const KEYTAR_SERVICE = 'MYC_Signer';
```

Comment:
Maybe use the full application name here, as this may be shown in GUI applications reading from the OS key chain (like the built-in macOS app).